gitlab_test.go

```diff
@@ -0,0 +1,266 @@
+package reviewdog
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/xanzy/go-gitlab"
+	"net/http"
+	"encoding/json"
+	"net/http/httptest"
+	"github.com/kylelemons/godebug/pretty"
+)
+
+const noGitlabTokenSkipTestMes = "skipping test (requires actual Personal access tokens. export REVIEWDOG_TEST_GITHUB_API_TOKEN=<GitLab Personal Access Token>)"
+
+func setupGitLabClient() *gitlab.Client {
+	token := os.Getenv("REVIEWDOG_TEST_GITLAB_API_TOKEN")
+	if token == "" {
+		return nil
+	}
+	cli := gitlab.NewClient(nil, token)
+	cli.SetBaseURL("https://gitlab.com/api/v4")
+	return cli
+}
+
+func TestGitLabMergeRequest_Post(t *testing.T) {
+	t.Skip("skipping test which post comments actually")
+	client := setupGitLabClient()
+	if client == nil {
+		t.Skip(noGitlabTokenSkipTestMes)
+	}
+
+	// https://gitlab.com/nakatanakatana/reviewdog/pull/2
```

Comment:
[question]
btw, does this test work if API is set properly? the URL returns 404.

[optional]
it's really great if I can join to the test repository as collaborator or something.
can we do this?

Reply:
sorry. I remove this repository. my mistake.
I create repository again, and all merge request URL changed.

Reply:
> [optional]
> it's really great if I can join to the test repository as collaborator or something.
> can we do this?

OK.
Is your GitLab.com account  `haya14busa` ?

https://gitlab.com/haya14busa

Reply:
yes that's me!